src/components/TableFilterPopover.tsx

```diff
@@ -6,13 +6,16 @@ import Popover from "@leafygreen-ui/popover";
 import TextInput from "@leafygreen-ui/text-input";
 import { size } from "constants/tokens";
 import { useOnClickOutside } from "hooks";
-import { useUpdateURLQueryParams } from "hooks/useUpdateURLQueryParams";
 
 const { gray, focus } = uiColors;
 
-export const JobLogsPopover: React.FC = () => {
-  const updateQueryParams = useUpdateURLQueryParams();
+interface TableFilterPopoverProps {
+  onConfirm: (filter: string) => void;
+}
 
+export const TableFilterPopover: React.FC<TableFilterPopoverProps> = ({
```

Comment:
It may make more sense to call this `TableSearchPopover` since it's for text search with a magnifying glass icon. I anticipate we'll have a more traditional "filter" component that uses a filter icon and checkboxes.

Reply:
true!!! i'll rename it 🔍